framework-plugins/lisk-framework-dashboard-plugin/src/plugin/dashboard_plugin.ts

```diff
@@ -73,6 +74,14 @@ export class DashboardPlugin extends BasePlugin {
 			defaults.config.default,
 			this.options,
 		) as dashboardPluginOptions;
+		const config = {
+			applicationUrl: this._options.applicationUrl,
+		};
+		// Write config file for faucet
+		writeFileSync(
+			join(__dirname, '../../build', 'config.js'),
+			`window.DASHBOARD_CONFIG = ${JSON.stringify(config)}`,
+		);
 		const app = express();
 		app.use(express.static(join(__dirname, '../../build')));
```

Comment:
As we already running express server, so we don't need to write to a static file. The configuration can be simply served from an express endpoint. Just register an endpoint to express `/config.json` and return all the configuration. Load it into React where needed.

Reply:
so you mean instead of `app.use(express.static(join(__dirname, '../../build')));` we use `app.use('/api/config', () => ({url : 'xxx' }))` and on UI, call `localhost:4005/api/config`?